REQUIRE

```diff
@@ -1,3 +1,3 @@
-julia 0.6
-DataFrames 0.11
-Compat 0.57.0
+julia 0.7
+DataFrames 0.13
+Compat 1.0
```

Comment:
Can't we get rid of the dependency on Compat?

Reply:
removed